src/components/TeamSection/TeamInformation.ts

```diff
@@ -15,6 +14,19 @@ import naman_jpg from "../../images/team/Naman_Bhoj.jpg";
 import lujaina_jpg from "../../images/team/Lujaina_ Eldelebshany.jpg";
 import hamza_jpg from "../../images/team/Hamza_Afzaal.jpg";
 import joel_jpg from "../../images/team/Joel_Happ.jpeg";
+import rachel_jpg from "../../images/team/Rachel_Renegado.jpg";
+import brian_jpg from "../../images/team/Brian_Nguyen.jpg";
+import esther_jpg from "../../images/team/Esther_Thompson.jpg";
+import ivan_jpg from "../../images/team/Ivan_Imshenetskyy.jpg";
+import mehrnaz_jpg from "../../images/team/Mehrnaz_Zafari.jpg";
+import parsa_jpg from "../../images/team/Parsa_Kargari.jpg";
+import youssef_jpg from "../../images/team/Youssef_Hamed.jpg";
+import teamData from "./teamMembers.json";
```

Comment:
This should be enough, and all the jpg imports above this line could be removed as you're importing it here instead :)
So just keep `import teamData from "./teamMembers.json";`

You can delete
```
import tyler_jpg from "../../images/team/Tyler_Chan.jpg";
import niyousha_jpg from "../../images/team/Niyousha_Raeesinejad.jpg";
import carol_jpg from "../../images/team/Carol_Wang.jpg";
import rajpreet_jpg from "../../images/team/Rajpreet_Gill.jpg";
import labib_jpg from "../../images/team/Labib_Ahmed.jpg";
import ana_jpg from "../../images/team/Ana_Laura.jpg";
import estefy_jpeg from "../../images/team/Estefy_Caballero.jpg";
import maheen_jpg from "../../images/team/Maheen_Raza.jpg";
import ben_jpg from "../../images/team/Ben_Schmidt.jpg";
import aarsh_jpg from "../../images/team/Aarsh_Shah.jpg";
import hilton_jpg from "../../images/team/Hilton_Luu.jpg";
import wilbur_jpg from "../../images/team/Wilbur_Elbouni.jpg";
import naman_jpg from "../../images/team/Naman_Bhoj.jpg";
import lujaina_jpg from "../../images/team/Lujaina_ Eldelebshany.jpg";
import hamza_jpg from "../../images/team/Hamza_Afzaal.jpg";
import joel_jpg from "../../images/team/Joel_Happ.jpeg";
import rachel_jpg from "../../images/team/Rachel_Renegado.jpg";
import brian_jpg from "../../images/team/Brian_Nguyen.jpg";
import esther_jpg from "../../images/team/Esther_Thompson.jpg";
import ivan_jpg from "../../images/team/Ivan_Imshenetskyy.jpg";
import mehrnaz_jpg from "../../images/team/Mehrnaz_Zafari.jpg";
import parsa_jpg from "../../images/team/Parsa_Kargari.jpg";
import youssef_jpg from "../../images/team/Youssef_Hamed.jpg";
```

Reply:
Oh I was going to remove the imports then I forgot. 
Wait, I shouldn't remove all the jpg imports?

Reply:
Sorry, GitHub PR comments are a bit weird. I meant remove all jpg imports above the line `import teamData from "./teamMembers.json";`. And make sure to test it